src/api/routes.ts

```diff
@@ -106,6 +106,10 @@ export function publishPackage(req: auth.AuthRequest, res: express.Response): vo
     });
 }
 
+export function whoami(req: auth.AuthRequest, res: express.Response): express.Response {
+  return res.status(200).json({ "username": res.locals.remote_user.name });
```

Comment:
My other way to do that was with a token. 
In `auth.ts` there is a function `middleware`, where we set 
`remote_user` propertie.